library/cache/src/main/java/com/gtc/getcamp/cache/LocalCacheProvider.kt

```diff
@@ -0,0 +1,10 @@
+package com.gtc.getcamp.cache
+
+import androidx.datastore.preferences.core.Preferences
+
+interface LocalCacheProvider {
+
+    suspend fun <T> storeValue(key: Preferences.Key<T>, value: T)
```

Comment:
its better that key value does not use dataStore library. because in this case we have to add datastore library to places that we use this interface.